microsoft-edge/webview2/how-to/set-preview-channel.md

```diff
@@ -45,9 +45,9 @@ When a WebView2 is initialized, it will attempt to find a valid runtime on the m
 
 There are several ways to make your WebView2 app use a specified preview channel of Microsoft Edge:
```

Comment:
update date field in YAML metadata at top

low pri, mostly a dummy comment to add self as reviewer

Reply:
Thanks, updated!